src/main/scala/com/github/unchama/seichiassist/subsystems/gacha/domain/GachaDrawSettings.scala

```diff
@@ -0,0 +1,31 @@
+package com.github.unchama.seichiassist.subsystems.gacha.domain
+
+import cats.effect.Sync
+import cats.effect.concurrent.Ref
+import cats.implicits._
+
+class GachaDrawSettings[F[_]: Sync] {
+
+  import ConsumeAmount._
+
+  val consumeGachaTicketAmountReference: Ref[F, ConsumeAmount] =
```

Comment:
```suggestion
  private val consumeGachaTicketAmountReference: Ref[F, ConsumeAmount] =
```

`consumeGachaTicketAmountReference`は`private`であって欲しいです。